CollapseLauncher/Program.cs

```diff
@@ -201,6 +204,8 @@ public static void Main(params string[] args)
                 MainEntryPointExtension.XamlCheckProcessRequirements();
                 ComWrappersSupport.InitializeComWrappers();
 
+                SystemEvents.SessionEnding += SystemEvent_EndingSession;
+
```

Comment:
As per Microsoft's .NET Runtime source-code, SystemEvents.SessionEnding is pretty much a wrapper to a call for ``WM_QUERYENDSESSION``. So I think we don't need to attach this and instead using the one from WndProc for invoking the StartBlocking

Reply:
![image](https://github.com/user-attachments/assets/8294c987-dade-48a2-a0e5-3cdf2c3f4494)
https://github.com/dotnet/runtime/blob/aad452c4b3ab9e525edd90de0b57aafd8b6bb80b/src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs#L909

Reply:
Oh yeah, the ``e.Cancel = false;`` is equivalent as returning 0/false from the WndProc